src/sentry/templates/sentry/emails/issue_alert_base.html

```diff
@@ -136,10 +136,12 @@ <h3>
                 {% else %}
                   <div class="event-type default">
                     <h3>
-                      <a href="{% absolute_uri link %}">{{ event.title|truncatechars:40 }}</a><br />
+                      <a href="{% absolute_uri link %}">{{ issue_type|truncatechars:40 }}</a>
```

Comment:
This changes how error alerts display at the moment. Just wanted to make sure that it's intentional (diff is in the visual snapshot)